x/qgb/orchestrator/orchestrator.go

```diff
@@ -374,7 +376,10 @@ func (orch Orchestrator) ProcessDataCommitmentEvent(
 	return nil
 }
 
-const DEFAULTCELESTIAGASLIMIT = 100000
+const (
+	DEFAULTCELESTIAGASLIMIT = 100000
+	DEFAULTCELESTIATXFEE    = 100
```

Comment:
Any reason we aren't being explicit with the types here?

Reply:
Merging, if this is needed, I can address it in a  separate PR.